generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management.state.ts

```diff
@@ -61,7 +67,7 @@ export const <%= entityInstance %>State = {
         {
             token: 'translate',
             deps: [JhiLanguageService],
-            resolveFn: (languageService) => languageService.setLocations(['<%= i18nToLoad %>'])
+            resolveFn: (languageService) => languageService.setLocations(<%- toArrayString(i18nToLoad) %>)
```

Comment:
Yes there is if you don't put the '-', the quotes in the array will be "&34" something instead of "'" (a proper quote).